app/components/ActivityVolonteersList.jsx

```diff
@@ -0,0 +1,100 @@
+var React = require('react')
+var AutoSuggest = require('react-autosuggest');
+
+
+var AutoSuggestVolonteer = React.createClass ({
+  getInitialState: function () {
+      return {volonteerId: 0};
+  },
+  getSuggestions: function(input, callback) {
+    var allVolonteers = this.props.allVolonteers;
+    
+    var words = input.split(" ");
+    var firstWordRegExp = new RegExp(words[0], 'i');
+    var  secondWordRegExp;
+    if (words[1]) { secondWordRegExp = new RegExp(words[1], 'i') };
+    
+    var suggestions = allVolonteers.filter(function (volonteer) {
+        if (words.length == 1) {
+            return firstWordRegExp.test(volonteer.first_name) || firstWordRegExp.test(volonteer.last_name);
+        } else if (words.length == 2) {
+            return (firstWordRegExp.test(volonteer.first_name) && secondWordRegExp.test(volonteer.last_name)) ||
+                   (secondWordRegExp.test(volonteer.first_name) && firstWordRegExp.test(volonteer.last_name));
+        }
+    });
+    
+    setTimeout(callback(null, suggestions.slice(0,5)), 300);
+  },
+  renderSuggestion: function (suggestion, input) {
+    return (
+        <span>{suggestion.first_name} {suggestion.last_name}</span>
+    );
+  },
+  getSuggestionValue: function (suggestionObj) {
+    this.setState({volonteerId: suggestionObj.id});
+    var output = suggestionObj.first_name+" "+suggestionObj.last_name; 
+    return output;
+  },
+  onClick: function () {
+    this.props.onAddButtonClick(this.state.volonteerId);
+  },
+  render: function () {
+    return (
+      <div>
+      <input type="button" onClick={this.onClick} value="Dodaj" />
+      <AutoSuggest
+                  id="activeVolonteers" 
+                  suggestions={this.getSuggestions} 
+                  suggestionRenderer={this.renderSuggestion}
+                  suggestionValue={this.getSuggestionValue}/>
```

Comment:
Ok, dzięki,
To żeby wczytać dane do bazy potrzebuję użyć skryptu https://github.com/Krakow2016/river . Tam jest w `localhost:9200/sdm/volounteer/_bulk`. Czy to jest nadal aktualne?